### ISSUE TITLE
vite v5.0.11 building for production failed 

### ISSUE THREAD

**Issue by tamer73** [laravel/vite-plugin#281]
### Vite Plugin Version

1.0.1

### Laravel Version

10.10

### Node Version

18.8

### NPM Version

8.1.8

### Operating System

Linux

### OS Version

5.15.114-gardenlinux-cloud-amd64 #1 SMP Debian 5.15.114-0gardenlinux3 (2023-06-01) x86_64 GNU/Linux

### Web browser and version

120.0.6099.216

### Running in Sail?

No

### Description

vite v5.0.11 building for production failed while installing Laravel Jetstream.

Source: https://jetstream.laravel.com/installation.html#install-jetstream-with-livewire

### Steps To Reproduce

Strictly following the installation guide from:
[https://jetstream.laravel.com/installation.html](https://github.com/laravel/jetstream/issues/url)

until:
php artisan jetstream:install livewire --teams

produces the given issue

**Comment by tamer73** [comment]
Which further info do you need? Node.js is installed over web management tool from provider Mittwald

**Comment by driesvints** [comment]
@tamer73 waiting until @jessarcher has time to look into this.

**Comment by timacdonald** [comment]
@tamer73 I've just run the following installation steps:

```
composer create-project laravel/laravel example-app
cd example-app
composer require laravel/jetstream
php artisan jetstream:install livewire --teams
```

And everything works as expected. Could you please provide the error message and the *exact* steps required to produce the error in an isolated way?

**Comment by jessarcher** [comment]
@timacdonald Error message is at https://github.com/laravel/jetstream/issues/1430. Doesn't look like it's related to this plugin. I'm assuming it's Rollup as it's the only thing I'm aware of in the stack that seems to use any Rust.

**Comment by timacdonald** [comment]
Ahh, I see.

In that case I will close this. This plugin is miles away from Rust, so if there is a bug it seems like it is upstream.

**Comment by tamer73** [comment]
Seems like missing libraries from my provider. I will inform you as soon i get more infos

**Comment by StrahinjaG** [comment]
If anyone else is searching for a solution, I refer to this thread: https://github.com/rollup/rollup/issues/5304. After a fresh installation of Laravel Breeze with Inertia, I encountered the same error message at Mittwald. I added 

`
"resolutions": {
"rollup": "npm:@rollup/wasm-node"
}
`

 to the end of my package.json and ran yarn install. After that, npm run build worked. I hope this helps someone.
